spec/controllers/api/v1/ratings_controller_spec.rb

```diff
@@ -14,13 +13,25 @@
 
   describe "show" do
     it "returns a rating" do
-    rating = create(:rating)
+      rating = create(:rating)
+
+      get :show, format: :json, id: rating.id
+
+      json_response = JsonResponse.new(response)
+      expect(json_response.headers).to have_http_status(:ok)
+      expect(json_response.rating_id).to eq(rating.id)
+    end
+  end
 
-    get :show, format: :json, id: rating.id
+  describe "create" do
+    it "creates a rating" do
+      user = create(:user)
+      skill = create(:skill)
+      allow_any_instance_of(Api::V1::BaseController).to receive(:current_user).and_return(user)
 
-    json_response = JsonResponse.new(response)
-    expect(json_response.headers).to have_http_status(:ok)
-    expect(json_response.rating_id).to eq(rating.id)
+      expect {
+        post :create, format: :json, rating: { "skill" => skill.id, "score" => 1 }
+       }.to change{ Rating.count }.by(1)
```

Comment:
`end` at 34, 7 is not aligned with `expect {` at 32, 6<br>Space missing to the left of {.